docs/_sidebar.md

```diff
@@ -1,6 +1,8 @@
 -   [**Getting Started**](getting-started.md)
 -   [**Installation**](installation.md)
 -   [**Bootstrapping**](bootstrapping.md)
+-   [**Migration**](migration)
```

Comment:
This sidebar isn't used, you need to add it to the sidebar in the developer portal!

So after merging this PR you'd need to merge the developer portal PR as well to get it to show up.

Reply:
Ah okay, thanks for the reminder!

Reply:
Added to the side bar in this PR: https://github.com/dhis2/developer-portal/pull/167 (currently commented; I guess it will wait on this one to be merged)